### DISCUSSION TITLE
Welcome to Resonate Discussions!

### DISCUSSION THREAD

**Discussion by chandansgowda** [AOSSIE-Org/Resonate#162] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Dante291** [comment]
@chandansgowda There is no installation guide for the project, it would really be appreciated if you could guide me how to install the project I mean the backend part.

**Reply by chandansgowda** [reply]
We are still working on the documentation. Consider joining our discord server for help regarding env setup.

**Reply by Aarush-Acharya** [reply]
Hey @chandansgowda  I would like to work on that

**Reply by Aarush-Acharya** [reply]
Will Insure that environment set up Guide is there and up by tomorrow

**Comment by ManasParab** [comment]
@Aarush-Acharya @chandansgowda i have been trying to setup the flutter app on my device and along with the backend. The docker setup is successful but the flutter app has the following error constantly, and i have no clue why i tried upgrading the package, removing the line of the code where error is occurring, and i even tried finding a solution online, via AI, etc. but couldnt fix it. Please help me resolve this issue.

![Client Error](https://github.com/user-attachments/assets/d0b0509c-1ceb-43b3-854e-c083adca6965)
![Client Error 2](https://github.com/user-attachments/assets/c08ca0e4-0527-4edb-b947-c4a7762d3633)
![Client Error 3](https://github.com/user-attachments/assets/2ac91dcd-d539-4da1-843c-ff2084f6708d)

**Reply by ManasParab** [reply]
Hey @Aarush-Acharya @chandansgowda can y'all please help me in this, as I am still not able to overcome and resolve this issue.

**Comment by Sanskriti10247** [comment]
@chandansgowda Hello! I’m a new contributor and I’ve just started getting familiar with this project. I’m very interested in contributing consistently and also hope to apply for GSoC 2026. May I kindly ask if this project is expected to be proposed for GSoC next year? Thanks a lot for maintaining this project and for your support!

**Comment by prempalhade3-code** [comment]
Hello everyone! I’m Prem Palhade, a new contributor here, and I’ve been actively exploring Resonate for about a week now. I’ve completed the setup, spent time understanding how the project is structured, and started following the ongoing discussions and workflows. Really appreciate the openness of this community—looking forward to learning more and contributing meaningfully.
